.circleci/config.yml

```diff
@@ -431,7 +434,14 @@ jobs:
           command: composer test-integration -- --log-junit test-results/php-integration/results.xml
       - run:
           name: Run << parameters.integration_testsuite >> integration test
-          command: DD_TRACE_AGENT_TIMEOUT=2000 composer << parameters.integration_testsuite >>
+          command: DD_TRACE_AGENT_TIMEOUT=2000 DD_TRACE_TEST_SAPI=<< parameters.sapi >> composer << parameters.integration_testsuite >>
```

Comment:
Do we have memory of why this 2000 timeout was necessary? If not I would take the opportunity to remove it as I believe it is not necessary.

Reply:
IIRC, this is set to a high limit because the `request-replayer` uses the built-in web server from the CLI SAPI which is way slower than the real Agent and so tests were regularly failing intermittently. I'll bump this down to 1000, but I think it still needs to remain pretty high until we make `request-replayer` faster.

I'll also break out those other tests into their own job. :)